### PR TITLE
Factorio port fixer

### LINKED ISSUE #15: Public browser listing shows the server but we arent able to connect.
The server now shows in public browser, however, when attempting to connect through the public browser you will get an error, the  server is available on the web and allows inbound connections when connecting to the ip directly. This is due to the fact the outbound ports are randomized and because the server needs to reach out to update the browser with connection details and keepalive state on port 34197. 

This is better explained by this writeup on [serverfault](https://serverfault.com/questions/1067199/routing-inbound-and-outbound-udp-traffic-to-the-same-port-in-kubernetes?newreg=227d674135a646f39cb9e70ab1e29f92); 

If anyone has any ideas on how we can look to resolve this I would be happy to implement them.

### PR DESCRIPTION
**Pull request by Kariton**
fixes #15 
closes: #35

### CODE REVIEW

**Comment by Kariton** [comment]
needs a bit of further testing.

**Comment by Kariton** [comment]
> Error: Service "factorio-server-charts-ohxmzmyutp" is invalid: spec.ports[0].targetPort: Invalid value: "31497": must contain at least one letter (a-z)

what? :D

**Comment by Kariton** [comment]
as known:
> cli: execute: auth: rcon: dial tcp 10.96.224.36:30100: connect: connection refused

**Review by SQLJames — APPROVED** [review]

File: charts/factorio-server-charts/values.yaml [hunk 1]
```diff
@@ -290,6 +296,29 @@ server_settings:
   maximum_segment_size: 100
   maximum_segment_size_peer_count: 10
 
+## @section Port-Fixer Parameters
+## @param port_fixer.enabled Enable the Factorio Port Fixer
+## @param image.repository Factorio port-fixer image repository
+## @param image.tag Factorio port fixer image tag (immutable tags are recommended)
+## @param image.pullPolicy Factorio port fixer image pull policy
+## @param image.port Factorio port to announce via port-fixer, defaults to service.port
+port_fixer:
```
Comment (MOZGIII) [inline charts/factorio-server-charts/values.yaml:305]: Should be `portFixer`?

**Pull request merged**